80s Game/Assets/Scripts/AI/StateMachine/ModifierBat/ModifierEffect.cs

```diff
@@ -34,15 +42,15 @@ void Update()
         // Manage duration timer if active
         if(bIsActive)
         {
+            Debug.Log(effectDuration);
```

Comment:
I believe this Debug Log might be causing some performance issues on my end when testing the game in editor